runtime/device/android/wenet/README.md

```diff
@@ -25,3 +25,31 @@ then just build and run the APK. Here is a gif demo, which shows how our on-devi
 Please note the wifi and data has been disabled in the demo so there is no network connection ^\_^.
 
 ![Runtime android demo](../../../../docs/images/runtime_android.gif)
+
+## Compute the RTF
+
+You could build the binary and the APK with Android Studio directly, or with the commands as follows:
```

Comment:
It's better we make the logic explicit by First, Second, Third, or Step 1, Step 2, Step 3

Reply:
Maybe step 1, step 2, step 3 is better

Reply:
Done.